README.md

```diff
@@ -11,10 +11,10 @@ module "s3" {
   source = "coralogix/aws/coralogix//modules/s3"
 
   coralogix_region   = "Europe"
-  CustomDomain       = "https://<your custom doamin>/api/v1/logs"
+  custom_domain       = "https://<your custom doamin>/api/v1/logs"
```

Comment:
If you have coralogix_region configured no need for custom_domain and vise-versa